cmd/juju/application/config.go

```diff
@@ -47,6 +47,20 @@ option and will output the value as plain text. This is provided to support
 scripts where the output of "juju config <application name> <setting name>" 
 can be used as an input to an expression or a function.
 
+Rather than specifying each setting name/value inline, the --file flag option
+may be used to provide a list of settings to be updated as a yaml file. The
+yaml file contents must include a single top-level key with the application's
+name followed by a dictionary of key/value pairs that correspond to the names
+and values of the settings to be set. For instance, to configure apache2,
+the following yaml file can be used:
+
+apache2:
+  servername: "example.com"
+  lb_balancer_timeout: 60
+
+The entire set of available config settings can be listed by running
```

Comment:
There was a move to update examples in help msg, with a description for each example. 
https://github.com/juju/juju/blob/87f9d9574e6ff7ac87a94ab685e1854c1d67a9fc/cmd/juju/application/deploy.go#L457

Looks like config didn't get changed yet, but since you're here.  Add descriptions the rest as a drive by?   And you move this description to the first example, ln 65.